src/Migrate/custom/Start-AzMigrateTestMigration.ps1

```diff
@@ -161,6 +164,7 @@ function Start-AzMigrateTestMigration {
                     }
                 }
             }
+            Set-AzContext -SubscriptionId $SourceSubscriptionId -ErrorVariable notPresent -ErrorAction SilentlyContinue
```

Comment:
>Set-AzContext [](http://example.com/codeflow?start=12&length=13)

If a failure occurs we wont be able to change context, can this cause issue?

Reply:
We need to change context to older context for failed case.